### ISSUE TITLE
Trivy dependency not found due to `PATH` override

### ISSUE THREAD

**Issue by Gowiem** [trunk-io/plugins#324]
## Issue

When running `trivy` through trunks plugin, I'm getting a failure on all files due to trivy hitting problems surrounding being invoked. Here is the copy / pasta of the stderr: 

```
        2023-06-21T14:06:56.666-0600	[34mINFO[0m	Need to update DB
        2023-06-21T14:06:56.666-0600	[34mINFO[0m	DB Repository: ghcr.io/aquasecurity/trivy-db
        2023-06-21T14:06:56.666-0600	[34mINFO[0m	Downloading DB...
        2023-06-21T14:06:56.666-0600	[31mFATAL[0m	init error: DB error: failed to download vulnerability DB: database download error: OCI repository error: 1 error occurred:
        	* error getting credentials - err: exec: "docker-credential-osxkeychain": executable file not found in $PATH, out: ``
```

This is due to trunk setting the `PATH` to only the linter in the `environment` as seen here: https://github.com/trunk-io/plugins/blob/main/linters/trivy/plugin.yaml#L56-L58

This seems to be an artifact of trivy having an implicit dependency on `docker-credential-osxkeychain`. 

I have the tool on my system, but it is not `PATH` set by trivy. 

See screenshot of the full error report: 

![CleanShot 2023-06-21 at 14 21 35](https://github.com/trunk-io/plugins/assets/1392040/ca5fb84c-29e7-4cba-a19d-bfb966c1946d)

## Expected

I would expect trunk to either download *ALL* dependencies for their plugins so they run in an isolated fashion. This would be preferred and would get around this issue. 

The other option is to provide a workaround so users like myself can better override the `environment` / `PATH` set for the plugin. That would allow me to specify the correct PATH so that trivy can execute as part of trunk. This has major issues as it is likely that setting the PATH in something like `trunk.yaml` would not be portable to other team members machines and would be fragile. 

## Notes

If we have a suggested plan of action for this issue, then I'd be happy to implement it as I don't believe the fix should be a major one. [Maybe adding an additional `downloads` entry](https://github.com/trunk-io/plugins/blob/main/linters/trivy/plugin.yaml#L3) for the [docker-credential-osxkeychain](docker-credential-osxkeychain) tool so that is installed alongside trivy?

**Comment by TylerJang27** [comment]
Hi! Thanks for reporting this; the very detailed breakdown and output is very helpful. Fortunately, we have a fix in the works for this bug and hope to have a release out as soon as possible!

**Comment by TylerJang27** [comment]
We just released plugins version 0.0.19, which includes a fix for this by including the user's path. If you're still running into any issues please let us know!
